src/Endpoints/Payments.php

```diff
@@ -119,6 +119,21 @@ public function create($data)
         return new Payment($res->json);
     }
 
+    /**
+     * @param string $id  The ID of the payment to cancel
+     *
+     * @return void
+     * @throws RequestError
+     */
+    public function cancel($id)
+    {
+        $res = $this->request(self::PAYMENTS_PATH . "/$id/cancel")->put();
```

Comment:
I saw everywhere stuff like " self::PAYMENTS_PATH . "/$id/cancel" " but it is not a good practice to build url we shoult use at least sprintf

Reply:
yes, it is a correction to be expected,everywhere at the same time